core/src/main/java/com/scalar/db/transaction/singlecrudoperation/SingleCrudOperationTransactionAdmin.java

```diff
@@ -0,0 +1,156 @@
+    distributedStorageAdmin.truncateTable(namespace, table);
+  }
+
+  @Override
+  public void createIndex(
+      String namespace, String table, String columnName, Map<String, String> options)
+      throws ExecutionException {
+    distributedStorageAdmin.createIndex(namespace, table, columnName, options);
+  }
+
+  @Override
+  public void dropIndex(String namespace, String table, String columnName)
+      throws ExecutionException {
+    distributedStorageAdmin.dropIndex(namespace, table, columnName);
+  }
+
+  @Override
+  public TableMetadata getTableMetadata(String namespace, String table) throws ExecutionException {
+    return distributedStorageAdmin.getTableMetadata(namespace, table);
+  }
+
+  @Override
+  public Set<String> getNamespaceTableNames(String namespace) throws ExecutionException {
+    return distributedStorageAdmin.getNamespaceTableNames(namespace);
+  }
+
+  @Override
+  public boolean namespaceExists(String namespace) throws ExecutionException {
+    return distributedStorageAdmin.namespaceExists(namespace);
+  }
+
+  @Override
+  public void importTable(String namespace, String table, Map<String, String> options)
+      throws ExecutionException {
+    distributedStorageAdmin.importTable(namespace, table, options);
+  }
+
+  @Override
+  public void repairTable(
+      String namespace, String table, TableMetadata metadata, Map<String, String> options)
+      throws ExecutionException {
+    distributedStorageAdmin.repairTable(namespace, table, metadata, options);
+  }
+
+  @Override
+  public void addNewColumnToTable(
+      String namespace, String table, String columnName, DataType columnType)
+      throws ExecutionException {
+    distributedStorageAdmin.addNewColumnToTable(namespace, table, columnName, columnType);
+  }
+
+  @Override
+  public Set<String> getNamespaceNames() throws ExecutionException {
+    return distributedStorageAdmin.getNamespaceNames();
+  }
+
+  @Override
+  public void repairNamespace(String namespace, Map<String, String> options)
+      throws ExecutionException {
+    distributedStorageAdmin.repairNamespace(namespace, options);
+  }
+
+  @Override
+  public void upgrade(Map<String, String> options) throws ExecutionException {
+    distributedStorageAdmin.upgrade(options);
+  }
+
+  @Override
+  public void createCoordinatorTables(Map<String, String> options) {
+    // Do nothing since the single CRUD operation transactions don't have coordinator tables
+  }
+
+  @Override
+  public void dropCoordinatorTables() {
+    // Do nothing since the single CRUD operation transactions don't have coordinator tables
+  }
+
+  @Override
+  public void truncateCoordinatorTables() {
+    // Do nothing since the single CRUD operation transactions don't have coordinator tables
+  }
+
+  @Override
+  public boolean coordinatorTablesExist() {
+    // Always return true since the single CRUD operation transactions don't have coordinator tables
```

Comment:
> since the single CRUD operation transactions don't have coordinator tables

In this case, I think returning true is a confusing behavior since it's not checked actually. How about throwing `UnsupportedOperationException`?

Regarding other empty methods (e.g., `createCoordinatorTables`), users might think the operation finished successfully if the API call finishes quietly. It might be better to throw `UnsupportedOperationException` as well when they are called?

Reply:
Good point. I didn't do that because if we do so, the application developer would need to switch the code depending on whether they are using Consensus Commit or not. To avoid this code switch, we always return `true` in `coordinatorTablesExist()` and do nothing in the other coordinator table-related methods. What do you think?

Reply:
That sounds reasonable. I also saw the same implementation in `JdbcTransactionAdmin`.

I'm okay with the current implementation as-is, but how about adding a warning message just in case?

```java
  @Override
  public boolean coordinatorTablesExist() {
    logger.warn("Always true is returned since the single CRUD operation transactions don't have coordinator tables");  
```

Reply:
Yeah, we can add warning messages for the coordinator table-related methods.

One thing I'm concerned about is that in ScalarDB Cluster, the cluster nodes try to create the coordinator tables if they don't exist. So, if we add warning messages, the messages are shown in the log of the cluster nodes, which might confuse users if they are using a non Consensus Commit transaction manager. What do you think?

Reply:
I see. My idea on the concern is as follows:

- With the single CRUD operation transaction mode, those Admin APIs for the coordinator table do nothing. A return value from the APIs doesn't make sense. So, the warning message should more emphasize that like this?
```java
    logger.warn("Always true is returned since the single CRUD operation transactions don't have coordinator tables. Please use this Admin API with Consensus Commit transaction mode if you want to get the actual state");
```
- ScalarDB Cluster nodes automatically create the coordinator table if not exists
- Users see the warning message after using the Admin APIs with the single CRUD operation transaction mode, but they understand the Admin APIs did nothing and the return value made no sense. So, they won't be surprised even if seeing the coordinator table exists

So, I don't think warning log confuses users. But I don't have a strong opinion on it. So, we can go as-is without waring log or throwing exception.

Reply:
Added Javadoc in aecdf951b623ea4d77be27435ca397eddb5414ce. Thanks.